### PR TITLE
Updated language in EvaChecks.csv from "Refused" to "Preferred Not To…

### PR DESCRIPTION
**Pull request by vlopez0603**
- Updated language in EvaChecks.csv from "Refused" to "Preferred Not To Answer."
- Also updated "refused" in helper_functions.R to "prefers not to answer."

### CODE REVIEW

File: public-resources/EvaChecks.csv
Comment (alex-silverman) [inline public-resources/EvaChecks.csv:1]: Only comment I would make is that we aren't being consistent between using "Client _**prefers**_..." and "Client _**preferred**_"

**Review by alex-silverman — COMMENTED** [review]
One small comment on tense - in one place we have **_prefers_** and in the other we have _**preferred**_

File: public-resources/EvaChecks.csv
Comment (vlopez0603) [reply public-resources/EvaChecks.csv:1]: This is a good point. The responses in the HMIS Data Standards are in the present tense (i.e. Client Doesn't Know, Client Prefers Not To Answer), so I would be open to going back and redoing these to stick with that tense.

Comment (vlopez0603) [reply public-resources/EvaChecks.csv:1]: Updated language and pushed the commit to this branch.

**Pull request merged**